src/widgets/label/lv_label.c

```diff
@@ -1281,8 +1281,8 @@ static void copy_text_to_label(lv_label_t * label, const char * text)
 #if LV_USE_ARABIC_PERSIAN_CHARS
```

Comment:
`LV_USE_ARABIC_PERSIAN_CHARS` is defined as 1 in the tests, maybe that's why the issue when its value is 0 wasn't found when executing the tests.